index.html

```diff
@@ -73,5 +61,138 @@ <h3 class="h3_title">I’am a Software Developer</h3>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+    </section>
+
+    <section class="section_aboutMe">
+      <h2 class="h2_4">About me</h2>
```

Comment:
- [ ] Kindly add some more space between the "See Project" button and the "About me" section to match with the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![a2](https://user-images.githubusercontent.com/70483835/159112312-da371fb3-8f63-4a99-bc65-b30c833c5d98.png) | ![a1](https://user-images.githubusercontent.com/70483835/159112327-4b2d656e-5b80-4a36-b7a3-2294db061b2b.png)

Reply:
**Thank you for this feedback**
I did all possible things to remove this horizontal scrollbar but I can't.
The body of my html has the 100vw Width and padding:0;  
So how can I remove the horizontal scrollbar?
I think it is a problem with Chrome not the code because in mozila there is not that horizontal scroll bar

Reply:
> **Thank you for this feedback** I did all possible things to remove this horizontal scrollbar but I can't. The body of my html has the 100vw Width and padding:0; So how can I remove the horizontal scrollbar? I think it is a problem with Chrome not the code because in mozila there is not that horizontal scroll bar

Reply:
I have checked it on mozila. The issue exists also there.

Reply:
I just fund were the issue was,

Reply:
Thank you !